### PR TITLE
chore(pubsub): release v1.5.0

### PR DESCRIPTION
**Pull request by hongalex**
This commit will be tagged v1.5.0

### CODE REVIEW

File: internal/version/version.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 
 // Repo is the current version of the client libraries in this
 // repo. It should be a date in YYYYMMDD format.
-const Repo = "20200701"
+const Repo = "20200706"
```
Comment (tbpg) [inline internal/version/version.go:29]: This seems to be an unrelated change?

Comment (codyoss) [reply internal/version/version.go:29]: [Step 7](https://github.com/googleapis/google-cloud-go/blob/master/RELEASING.md#how-to-release-a-submodule). Usually if the regen PR was merged that day, this step makes no difference.

File: pubsub/CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Changes
 
+## v1.5.0
+
+- Add support for subscription detachment
```
Comment (codyoss) [inline pubsub/CHANGES.md:5]: nit: add periods.
